tsconfig.json

```diff
@@ -15,7 +15,6 @@
 		"noUncheckedIndexedAccess": true,
 		"esModuleInterop": true,
 		"resolveJsonModule": true,
-		"skipLibCheck": true,
```

Comment:
[suggest] Can you change `true` to `false` to make it easier for us to find a commit history?

```diff
-"skipLibCheck": true,
+"skipLibCheck": false,
```